blueprints/node-webkit/files/nw-runner.js

```diff
@@ -0,0 +1,35 @@
+var exec = require('child_process').exec;
+var NW_VERSION = '0.12.0';
+var os = require('os');
+var fs = require('fs');
+var findNw = require('./node_modules/ember-cli-node-webkit/lib/helpers/find-nw');
```

Comment:
Doesn't `require('ember-cli-node-webkit/lib/helpers/find-nw');` work? I think you can remove the `./node_modules` and reference this as you would any external NPM module. I'll let you confirm.